HAST-295: Open-source Hast.Core (#98)
Lombiq/Hastlayer-SDK: Docs/ReleaseNotes.md

## vNext

- The consumer sample app now uses [Terminal.Gui](https://github.com/migueldeicaza/gui.cs) to provide an easy to use GUI within a terminal window. Command-line configurability remains and is even extended.

> - Hastlayer is now full open-source! Formerly, `Hast.Core`, the core logic that did the .NET to hardware transformation was private. But now, the whole of Hastlayer is available for you in fully open form!

Reviewer: Lots of unnecessary commas make the sentence too fragmented.
```suggestion
- Hastlayer is now fully open-source! Previously `Hast.Core` (the core logic that did the .NET-to-hardware transformation) was private. But now the whole of Hastlayer is available for you as fully open source software!
```
